tests/core/tests/authentication.py

```diff
@@ -33,8 +32,8 @@ def test_is_authenticated(self):
         auth = Authentication()
         request = HttpRequest()
         # Doesn't matter. Always true.
-        self.assertTrue(auth.is_authenticated(None))
-        self.assertTrue(auth.is_authenticated(request))
+        self.assertEqual(auth.is_authenticated(None), True)
```

Comment:
Id' be in favor if making it return either True or False, it's a common point of confusion for new users. If we put it in a minor release instead of a micro release it should be fine.